pkg/module/scaffold/init.go

```diff
@@ -16,10 +16,7 @@ const directoryMode os.FileMode = 0755
 func createResources(rb resourceBuilder) error {
 	//paths may be absolute, they are resolved against the current working directory anyway
 	return rb.createDirectory("/charts").
-		createDirectory("/crds").
 		createDirectory("/operator").
```

Comment:
this as well?

Reply:
No, the operator folder stays in the layout to store the operator(s) source code. It was removed from the layering so it will not be present in the descriptor and image.